### PR TITLE
Plane: changes in Ardupilot Project introduction

### PR DESCRIPTION
**Pull request by Rajeta-Sarkar**
Made some changes in the introduction. Changes include removal of unnecessary comma and correcting the spelling mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,10 @@
 
 [![Autotest Status](https://autotest.ardupilot.org/autotest-badge.svg)](https://autotest.ardupilot.org/)
 
-Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
-been under development since 2010 by a diverse team of professional engineers, computer scientists and community contributors.
-Our autopilot software is capable of controlling almost any vehicle system imaginable, from conventional
-airplanes, quadplanes, multirotors, and helicopters, to rovers, boats, balancebots and even submarines. It is continually being expanded to provide
-support for new emerging vehicle types.
+Ardupilot is the most advanced, full-featured and reliable open source autopilot software available.
```
Comment (peterbarker) [inline README.md:17]:
```suggestion
ArduPilot is the most advanced, full-featured and reliable open source autopilot software available.
```

**Review by peterbarker — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tridge) [reply README.md:17]: @peterbarker feel free to fixup and merge

**Comment by tridge** [comment]
fine to merge with Peters fix

**Comment by peterbarker** [comment]
Merging, thanks @Rajeta-Sarkar

**Comment by Rajeta-Sarkar** [comment]
Thankyou for merging @peterbarker .

**Pull request merged**